SDF-to-Coords

```diff
@@ -0,0 +1,23 @@
+#Folder pathway: D:\SDFs\structures
+#This next bit of code is to see whether I can pull a file from the directory
+
+def SDF_to_coords(g):
+    path = "/mnt/d/SDFs/"
+    for g in path:
+        print(g)
+        
```

Comment:
Is your question that these lines don't work? If so, that's because that's not how you loop over files in a directory in Python. See [here](https://www.newbedev.com/python/howto/how-to-iterate-over-files-in-a-given-directory/) for how to do that. Also if this is a Python script it should have a `.py` extension (that'll trigger GitHub's syntax highlighting, which makes the file much easier to read).